### ISSUE TITLE
Glimpse-specific changes need to be translated

### ISSUE THREAD

**Issue by TrechNex** [glimpse-editor/Glimpse#326]
### Describe the bug
We have had to make changes throughout the application to use "Glimpse" instead of "GIMP" and our own copyright notices in the About window.

These things all need to be translated. Similarly, the reason why we use "Glimpse" instead of "Glimpse Image Editor" through the application is because the phrase "Image Editor" also needs to be translated!

### Screenshots
![ESJChjUW4AAj_2a](https://user-images.githubusercontent.com/45732696/75931339-99eac400-5e6c-11ea-827c-78cd0288f9d5.png)

### Desktop (please complete the following information)
 - OS: All

### Additional context
Original tweet: https://twitter.com/FastOSlinux/status/1234622404778954752

**Comment by TrechNex** [comment]
cc @dwkretz

**Comment by Member1221** [comment]
@TrechNex I personally think we can call Glimpse "Glimpse" in the application, that it's an image editor is more relevant on our website.

**Comment by TrechNex** [comment]
Indeed, I agree that where it's appropriate for different translations we're better off sticking to just "Glimpse".

The places I was thinking of was the splash screen title, text in **About** > **Help** and the main window title.

The reason we use "GNU I.M.P" liberally is because "GNU Image Manipulation Program" has exactly the same problem.

Also because our splash screen is a static image we're better off sticking with "Glimpse" there anyway.
